src/commands/cache/get/index.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023 Robert Bosch GmbH
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache License, Version 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Command } from '@oclif/core';
+import { mapReplacer } from '../../../modules/helpers';
+import { ProjectCache } from '../../../modules/project-cache';
+
+export default class Get extends Command {
+    static description = 'Get the cache contents as JSON string.';
+
+    static examples = [`$ velocitas cache get`];
+
+    async run(): Promise<void> {
```

Comment:
What do you think of adding the possibility to also get a cache value by key?

Reply:
Done